docs/user-guide/horizontal-pod-autoscaling/walkthrough.md

```diff
@@ -20,6 +21,12 @@ as Horizontal Pod Autoscaler uses it to collect metrics
 (if you followed [getting started on GCE guide](/docs/getting-started-guides/gce),
 heapster monitoring will be turned-on by default).
 
+In order to follow the sections on specifying multiple metrics, you will need a cluster and kubectl
+version 1.6 or later.  Furthermore, in order to make use of custom metrics, you will need to have
+properly configured your cluster to be able to communicate with the API server providing the custom
```

Comment:
"you will need to have properly configured your cluster to be able to communicate with the API server" -> "your cluster must be able to communicate with the API server..."